_delphi_utils_python/delphi_utils/nancodes.py

```diff
@@ -0,0 +1,19 @@
+"""Unified not-a-number codes for the indicators.
+
+Currently requires a manual sync between the covidcast-indicators
```

Comment:
this wont be true once we deploy to pypi right?

Reply:
Right, I can take it out.